lib/srv6.h

```diff
@@ -116,6 +119,19 @@ struct srv6_locator_chunk {
 	uint8_t proto;
 	uint16_t instance;
 	uint32_t session_id;
+
+	uint8_t flags;
+};
+
+enum srv6_endpoint_behavior_codepoint_t {
```

Comment:
For enums, we usually do not add `_t` suffix. More like for typedefs.

Reply:
Changed `srv6_endpoint_behavior_codepoint_t` to `srv6_endpoint_behavior_codepoint`.
Thanks for the suggestion.

I also added a comment to remind us that these values are the codepoints defined by IANA.